vcfa/resource_vcfa_org_networking.go

```diff
@@ -0,0 +1,148 @@
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/vmware/go-vcloud-director/v3/govcd"
+	"github.com/vmware/go-vcloud-director/v3/types/v56"
+)
+
+const labelVcfaOrgNetworking = "Organization Networking Settings"
+
+func resourceVcfaOrgNetworking() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceVcfaOrgNetworkingCreateUpdate,
+		ReadContext:   resourceVcfaOrgNetworkingRead,
+		UpdateContext: resourceVcfaOrgNetworkingCreateUpdate,
+		DeleteContext: resourceVcfaOrgNetworkingDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceVcfaOrgNetworkingImport, // The same as importing the Org
+		},
+
+		Schema: map[string]*schema.Schema{
+			"org_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: fmt.Sprintf("Parent %s for %s", labelVcfaOrg, labelVcfaOrgNetworking),
+			},
+			"log_name": {
+				Type:             schema.TypeString,
+				Required:         true,
+				Description:      fmt.Sprintf("A globally unique identifier (max 8 char) for this %s in the logs of the backing network provider", labelVcfaOrg),
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringLenBetween(0, 8)),
+			},
+			"networking_tenancy_enabled": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Whether this Organization has tenancy for the network domain in the backing network provider",
+			},
+		},
+	}
+}
+
+func resourceVcfaOrgNetworkingCreateUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	vcfaClient := meta.(*VCDClient)
+
+	org, err := vcfaClient.GetTmOrgById(d.Get("org_id").(string))
+	if err != nil {
+		return diag.Errorf("error retrieving %s: %s", labelVcfaOrg, err)
+	}
+
+	d.SetId(org.TmOrg.ID)
+	dSet(d, "org_id", org.TmOrg.ID)
+
+	orgNetworkingSettings, err := getTmOrgNetworkingSettingsType(vcfaClient, d)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	_, err = org.UpdateOrgNetworkingSettings(orgNetworkingSettings)
+	if err != nil {
+		return diag.Errorf("error updating %s for %s:%s", labelVcfaOrgNetworking, labelVcfaOrg, err)
+	}
+
+	return resourceVcfaOrgNetworkingRead(ctx, d, meta)
+}
+
+func resourceVcfaOrgNetworkingRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	vcfaClient := meta.(*VCDClient)
+	org, err := vcfaClient.GetTmOrgById(d.Get("org_id").(string))
+	if err != nil {
+		return diag.Errorf("error retrieving %s: %s", labelVcfaOrg, err)
+	}
+
+	d.SetId(org.TmOrg.ID)
+	dSet(d, "org_id", org.TmOrg.ID)
+
+	orgNetworkingSettings, err := org.GetOrgNetworkingSettings()
+	if err != nil {
+		if govcd.ContainsNotFound(err) { // Org no longer present, removing from state
+			d.SetId("")
+			return nil
+		}
+		return diag.Errorf("error retrieving %s for %s:%s", labelVcfaOrgNetworking, labelVcfaOrg, err)
+	}
+
+	err = setTmOrgNetworkingSettingsData(vcfaClient, d, orgNetworkingSettings)
```

Comment:
```suggestion
	err = setOrgNetworkingSettingsData(vcfaClient, d, orgNetworkingSettings)
```

Reply:
🤦‍♂️

Reply:
Fixed here and getOrgNetworkingSettingsType